core/create/src/index.ts

```diff
@@ -82,14 +82,29 @@ async function executeMigration(
       )
     : Promise.resolve()
 
+  const gitignorePath = '.gitignore'
+  const fixGitignoreAsync = async () => {
```

Comment:
**nitpick:** maybe we want to include a comment here as to why we are doing this? I was trying to think of a way we could do that via naming but think it's a little more involved than that

Reply:
do you mean document what we are doing in this closure (editing the `.gitignore`) or why we're creating an async closure in the first place (so that we can wrap all this logic in one komatsu log)?

Reply:
I mean the why!